src/SprykerEco/Shared/Heidelpay/HeidelpayConstants.php

```diff
@@ -50,4 +50,7 @@ interface HeidelpayConstants
     public const CONFIG_ENCRYPTION_KEY = 'HEIDELPAY:CONFIG_ENCRYPTION_KEY';
 
     public const CONFIG_IS_SPLIT_PAYMENT_ENABLED_KEY = 'HEIDELPAY:CONFIG_IS_SPLIT_PAYMENT_ENABLED_KEY';
+
+    public const CONFIG_EASY_CREDIT_LOGO_URL = 'HEIDELPAY:CONFIG_EASY_CREDIT_LOGO_URL';
```

Comment:
I think you don't need "CONFIG" word in name of these env constants.

Reply:
removed

Reply:
@Ruslan05 please revert your changes back. 
I agree with @aleksey-kotsuba that without *CONFIG_* is better but since we already have all configs in the module with this prefix we have to fix consistency.